### ISSUE TITLE
Manual selection mode

### ISSUE THREAD

**Issue by ST-DDT** [sullerandras/UtilizationMonitor#15]
Add a new run-mode.

* Off (don't track anything)
* **NEW:** Manual (only track selected entities )
* Automatic (track newly build entities)

Use a selection tool to add or remove tracked entities instead of auto tracking them.
This might be a good alternative for big factories, where only a new extension should be measured.

**Comment by ST-DDT** [comment]
I will look at that if I have some free time.

**Comment by ST-DDT** [comment]
@sullerandras What do you think about this feature?

**Comment by sullerandras** [comment]
Yes, this probably makes sense. Another possibility is to add all entities for tracking which are near the player instead of using a selection tool.
